packages/discord.js/src/structures/BaseGuild.js

```diff
@@ -59,18 +66,20 @@ class BaseGuild extends Base {
 
   /**
    * The acronym that shows up in place of a guild icon
+   *
    * @type {string}
    * @readonly
    */
   get nameAcronym() {
     return this.name
-      .replace(/'s /g, ' ')
-      .replace(/\w+/g, e => e[0])
-      .replace(/\s/g, '');
+      .replaceAll("'s ", ' ')
+      .replaceAll(/\w+/g, word => word[0])
+      .replaceAll(/\s/g, '');
   }
```

Comment:
This change seems a lot slower: https://jsbm.dev/6Ydo8tQAO21w4

Granted this likely is not used much or has a need to be performant. Still...

Reply:
Huh, you‘re right. Although I don’t see why, considering passing a global regex to replace and replaceAll should use the exact same algorithm…